packages/vue/src/composables/useAuth.ts

```diff
@@ -53,20 +52,25 @@ export const useAuth = createSharedComposable((): UseAuth => {
 export const useAuthenticator = createSharedComposable(() => {
   const { authStatus, state, send } = useAuth();
 
-  const useAuthenticatorValue = reactive({
-    ...facade,
-    send,
-    state,
-  }) as any;
+  // TODO(BREAKING): remove the cast to any
+  const useAuthenticatorValue = reactive({}) as any;
```

Comment:
This initial value is not needed, because `watchEffect` below is guaranteed to run immediately (see comment below). And `...facade` was spreading a mock value anyway, so we aren't losing anything.

I actually prefer this because now we only have one place where we populate `useAuthenticatorValue`.